tasks/request-and-upload-signature/request-and-upload-signature.yaml

```diff
@@ -0,0 +1,174 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: request-and-upload-signature
+spec:
+  description: >-
+    Tekton task to request and upload a simple signature.
+
+    - This task is meant to be used in an internal pipeline that can be triggered frequently
+      and is expected to complete as quickly as possible.
+
+  params:
```

Comment:
Can we get `type: string` on these? If we are missing it in lots of places, fine, but my best guess is we have it in most spots

Reply:
added type